src/wp-admin/update.php

```diff
@@ -218,65 +233,76 @@
 		iframe_footer();
 	} elseif ( 'install-theme' == $action ) {
 
-		if ( ! current_user_can('install_themes') )
+		if ( ! current_user_can( 'install_themes' ) ) {
 			wp_die( __( 'Sorry, you are not allowed to install themes on this site.' ) );
+		}
 
-		include_once( ABSPATH . 'wp-admin/includes/class-wp-upgrader.php' ); //for themes_api..
+		include_once ABSPATH . 'wp-admin/includes/class-wp-upgrader.php'; //for themes_api..
 
 		check_admin_referer( 'install-theme_' . $theme );
-		$api = themes_api('theme_information', array('slug' => $theme, 'fields' => array('sections' => false, 'tags' => false) ) ); //Save on a bit of bandwidth.
+		$api = themes_api(
+			'theme_information',
+			array(
+				'slug'   => $theme,
+				'fields' => array(
+					'sections' => false,
+					'tags'     => false,
+				),
+			)
+		); //Save on a bit of bandwidth.
```

Comment:
comment should be moved to a code line

Reply:
Odd but matches upstream:
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/update.php#L273

Reply:
same formatter, same results, but not a good way to leave it

Reply:
PR updated moving this comment up.